gavel/__init__.py

```diff
@@ -23,6 +23,12 @@
 )
 assets.register('scss_all', scss)
 
+from celery import Celery
+app.config['CELERY_BROKER_URL'] = 'redis://localhost:6379/0'
+app.config['CELERY_RESULT_BACKEND'] = 'redis://localhost:6379/0'
```

Comment:
Eventually, you should change this to read from `settings` (and then update the settings template and vagrant settings template accordingly). See line 10 in this file, for example.

Also, if these URLs are always the same (are they? idk), then you could just use a single setting for both.